<launch>
  <!-- params -->
  <param name="/pedsim/scene_file" value="$(find pedsim_simulator)scenarios/static_group_nav.xml" type="string"/>
  <param name="/pedsim/update_type" value="oracle" type="string"/>
  <param name="/pedsim/sensor_range" value="30.0" type="double"/>
  <param name="/pedsim/cell_size" value="1.0" type="double"/>


  <!-- other parameters -->
  <param name="/pedsim_simulator/max_robot_speed" value="2.0" type="double"/>
  <param name="/pedsim_simulator/robot_wait_time" value="3" type="double"/>
  <param name="/pedsim_simulator/teleop_flag" value="1.0" type="double"/>


  <!-- main simulator node -->
  <node name="simulator" pkg="pedsim_simulator" type="simulator" output="screen"/>

    <!-- tf broadcaster for aligning frames -->
  <node name="simulator_tf_broadcaster" pkg="pedsim_simulator" type="dummy_transforms.py" respawn="false" output="log"
/>


  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pedsim_simulator)/rviz/demo_formations.rviz"/>

  <!-- Keyboard teleoperation node -->
<node name="robot_teleop_keyboard" pkg="pedsim_simulator" type="robot_teleop_keyboard" respawn="false" output="screen" />


<node name="mocktracks_info_screen" pkg="pedsim_simulator" type="mocktracks_info_screen.py" respawn="false" output="screen" cwd="node"/>

<!--<node name="mockgroups_info_screen" pkg="pedsim_simulator" type="mockgroups_info_screen.py" respawn="false" output="screen" cwd="node"/>-->

</launch>
